packages/mini-apps-ui-kit-react/stories/BulletListItem.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import BulletPoint from "@/components/BulletPoint/BulletPoint";
```

Comment:
wdyt of making this story a `subcomponent` in the main `BulletList.stories.tsx`. It adds another tab to explore the props of subcomponents.